algo/src/main/resources/database/generate_serve_db.sql

```diff
@@ -14,6 +14,8 @@ CREATE TABLE `actionableHotspot`
     `sourceUrls` varchar(2000),
     `studyNctId` varchar(100),
     `studyTitle` varchar(500),
+    `studyAcronym` varchar(500),
```

Comment:
Minor: I'd assume the acronym would be much shorter, so varchar(500) could be much more than required. But if not sure of length standards for the acronym then this is safer as is.

Reply:
I think the length of this field can be indeed chorter. It is the acronym of the studyTitle. I have update it to varchar(250) becuae I am not sure what the required minimal varchar should be